config/staging/iatlas-runner.yaml

```diff
@@ -0,0 +1,16 @@
+template_path: gitlab-runner.yaml
+stack_name: iatlas-gitlab-runner-staging
+dependencies:
+  - common/iatlasvpc.yaml
+  - staging/iatlas-db.yaml
+  - staging/iatlas-kms.yaml
+stack_tags:
+  Department: "CompOnc"
+  Project: "iAtlas"
+  OwnerEmail: "andrew.lamb@sagebase.org"
+parameters:
+  InstanceType: "t2.nano"
```

Comment:
some of these are already set as defaults in the CFN template so do you need to put them here?  if you want to be explicit then its fine.

Reply:
Yes, just being explicit.